zerver/openapi/zulip.yaml

```diff
@@ -504,18 +504,30 @@ paths:
                                     - type: object
                                       additionalProperties: false
                                       description: |
-                                        When the delivery email of a user changes.
+                                        When the delivery email of a user changes or access to the delivery email
+                                        changes for you due to change in `email_address_visibility` setting.
 
-                                        Note: This event is only visible to admins.
+                                        **Changes**: Prior to Zulip 7.0 (feature level 162), this event was sent
+                                        only to the affected user, and this event would only be triggered by
+                                        changing the affected user's delivery email.
```

Comment:
I think this is inaccurate -- previously it would be sent both to the affected user and all organization admins, right?

Reply:
As per the code [here](https://github.com/zulip/zulip/blob/4dfe3d36e0c4373ea7f30daa894a4f195330e49c/zerver/actions/user_settings.py#L69), it was sent to affected user only. I guess the documentation here was incorrect, unless I am missing something.

Reply:
Cool, didn't know that :)